### PR TITLE
add a comment that you need to join the project to see the hangout button

### PR DESCRIPTION
**Pull request by piotrkedzia**
I modified views/project/show and partial that render hangout button.
New hangout button for users that no follow project works great but many cucumber tests are now failing.
In my opinion these failing test are no related to project/show view.
So Thomas if you have a moment please take a look on this.

https://www.pivotaltracker.com/story/show/76848756

### CODE REVIEW

**Comment by tochman** [comment]
We have failing tests on this one and we also need to test for the new behavior.

File: app/views/event_instances/_hangout_button.html.erb [hunk 1]
```diff
@@ -8,8 +9,32 @@
     'callbackUrl' => hangout_url('id').gsub(/id$/, '')
   }) %>
```
Comment (tochman) [inline app/views/event_instances/_hangout_button.html.erb]: I would suggest moving this out to a helper class. It feels like too much logic for the view. It was a cornercase before and it definitely now with this if condition.

**Comment by piotrkedzia** [comment]
Moved hangout button for no followers frow partial direct to project/show view.
Now tests should pass.

File: app/views/event_instances/_hangout_button.html.erb (hunk 1 above)
Comment (piotrkedzia) [inline app/views/event_instances/_hangout_button.html.erb]: Moved all changes from this partial to view. So this file is untouched by me right now.

**Comment by piotrkedzia** [comment]
I'll add cucumber tests as soon as posible.

**Comment by piotrkedzia** [comment]
Thomas: I added a test that we talked about during the Friday WSO meeting. I hope now it's ready to merge :)

**Comment by tochman** [comment]
Cool. Great work.

**Pull request merged**